src/serial-provision-dialog.ts

```diff
@@ -180,95 +202,130 @@ class SerialProvisionDialog extends LitElement {
     }
 
     const selectedSsid = this._ssids?.find(
-      (info) => info.name === this._selectedSsid
+      (info) => info.name === this._selectedSsid,
     );
 
     return html`
       <div>
         Enter the credentials of the Wi-Fi network that you want your device to
         connect to.
       </div>
+      ${this._client?.info ? this._renderDeviceInfo() : ""}
```

Comment:
Let's put this above the text on what to do with the form

Reply:
I'll take care of both of those.